pkg/adsc/delta_test.go

```diff
@@ -410,8 +409,8 @@ LDS/:
 			tracker := assert.NewTracker[string](t)
 			handlers := buildHandlers(t, tracker)
 
-			client := NewDeltaWithBackoffPolicy(l.Addr().String(), &DeltaADSConfig{}, nil, handlers...)
-			assert.NoError(t, client.Run(test.NewContext(t)))
+			client := NewDelta(l.Addr().String(), &DeltaADSConfig{}, handlers...)
+			go client.Run(ctx)
```

Comment:
any other place need to fix?

Reply:
I think the Delta is currently only used in test